### PR TITLE
Add cas studies from Poland (Softelnet)

### PR DESCRIPTION
**Pull request by yjacolin**
Do not merge, I need to add figures.

### CODE REVIEW

File: source/site/about/case_studies/poland_ffth.rst [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+Using QGIS for FTTH/GPON network planning due to the implementation European Digital Agenda
+===========================================================================================
+
+Definition of the problem
+==========================
+
+Over the past few years many EU countries have launched operational programs to
+implement European Digital Agenda (EAC) developed by the European Commission in
+2010. According to its guidelines until 2020 every citizen of the European Union
+should have access to 30 Mbps Internet bandwidth and at least half of the them
```
Comment (DelazJ) [inline source/site/about/case_studies/poland_ffth.rst]: half of ~the~ them

File: source/site/about/case_studies/poland_ffth.rst [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+* GRASS - a collection of vector and network algorithms,
+* QGIS plugins - OpenLayer, QuickOSM, Scipy Point Clustering, WorkContextTracer
+  (our plug-in),
+* QGIS modeler – tool for graphical flow modelling,
+* SpatiaLite - a local database for storing input and output data.
+
+The entire solution was based on the concept of so called wizards which realize
+the process of preparing the FTTx network technical concept step by step. Using
+'geoprocessing' modeller, we created models requiring only the necessary input
+data from operator.  Algorithmic complexity was encapsulated within models in
```
Comment (DelazJ) [inline source/site/about/case_studies/poland_ffth.rst]: an extra space before "Algorithmic"

File: source/site/about/case_studies/poland_ffth.rst [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+the form of designed flow, which use existing QGIS algorithms and calls
+dedicated scripts.
+
+To prevent the results of subsequent analyses from getting mixed up, we provided
+a plug-in called WorkContextTracer which groups our work in so called work contexts.
```
Comment (DelazJ) [inline source/site/about/case_studies/poland_ffth.rst]: rather **plugin**?

File: source/site/about/case_studies/poland_ffth.rst [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+   :scale: 60%
+   :align: left
+
+Network algorithms based on the graph functions available in GRASS were
+enriched by adding the telecommunication scripts. We introduced a concept of
+demand clustering which enables to use both K-means and hierarchical methods to
+aggregate demand points.
+
+Geoprocessing scripts using cable routing layer and Djikstra algorithm develops
+a cable route proposal with optical fiber nodes locations.
```
Comment (DelazJ) [inline source/site/about/case_studies/poland_ffth.rst]: i'm not sure I understand this sentence; any missing word?

File: source/site/about/case_studies/poland_ffth.rst [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+* http://www.ftthcouncil.eu/  - FTTH Council Europe website
+* https://www.scipy.org/ - SciPy project website
+* https://ksavinetworkinventory.com/en/download-geospatial-network-inventory-free/
+  - tool download webpage. 
+
+Authors
+=======
+
+The solution is a completely free open source tool. It is part of a larger
+project based on QGIS and FreeCAD platforms with has intention to be used in
```
Comment (DelazJ) [inline source/site/about/case_studies/poland_ffth.rst]: **which** has or with **the**?

**Review by DelazJ — COMMENTED** [review]
Thanks for pushing this text; i added some comments.

Very interesting use case. Might interest many users out there. btw, could be nice to have a channel (or a simple blog post) to inform users when a new case study is added to the website. Not sure many people notice these changes...

**Comment by yjacolin** [comment]
@DelazJ about your note on the blog note, we should store use case iun the website but that's doesn't meant that we don't need to share on twitter, QGIS linkedin group and blog post. What's your opinion @rduivenvoorde, can you write a blog post on this?

**Comment by DelazJ** [comment]
@yjacolin yes, my idea wasn't to move them out of the website. It's fine there. Just thinking that we could promote a bit this section of the website which is full of knowledge. Blog post, twitter or any other social media is good enough I think

**Comment by yjacolin** [comment]
ok, I can manage a post in the QGIS linkedin group :)

**Pull request merged**